src/TLuaInterpreter.h

```diff
@@ -406,11 +407,12 @@ class TLuaInterpreter : public QThread
     static int setDefaultAreaVisible(lua_State*);
     static int getProfileName(lua_State*);
     static int raiseGlobalEvent(lua_State*);
-    static int setServerEncoding(lua_State *);
-    static int getServerEncoding(lua_State *);
-    static int getServerEncodingsList(lua_State *);
+    static int setServerEncoding(lua_State*);
+    static int getServerEncoding(lua_State*);
+    static int getServerEncodingsList(lua_State*);
     static int alert(lua_State* L);
-// PLACEMARKER: End of Lua functions declarations
+    static int tempPromptTrigger(lua_State* L);
```

Comment:
💡 No need for an argument variable `L` I guess...?

Reply:
Oops yeah. It was autogenerated and I didn't notice.